lisa-kernel/src/main/scala/lisa/kernel/proof/RunningTheory.scala

```diff
@@ -131,42 +131,48 @@ class RunningTheory {
    * @param label          The desired label.
    * @param expression     The functional term defining the function symbol.
    * @param out            The variable representing the function's result in the formula
+   * @param proven         A formula possibly stronger than "expression" that the proof proves. It is always correct if it is the same as "expression", but
+   *                       if "expression" is less strong, this allows to make underspecifid definitions.
```

Comment:
`underspecifid -> underspecified`

Also might be better to use `expression` instead of "expression" since it refers to an object in code.